h/feeds/test/rss_feed_test.py

```diff
@@ -0,0 +1,160 @@
+import datetime
+
+import mock
+
+from h.test import factories
+from h.feeds import rss_feed
+
+def _annotation_url():
+    """Return a mock annotation_url() function.
+
+    It just returns a hard-coded URL, enough to make the code that calls this
+    function not crash.
+
+    """
+    return mock.Mock(return_value='https://hypothes.is/a/id')
+
+
+def test__feed_from_annotations_pubDate():
```

Comment:
You're testing a lot of private functions in this file. Is there something that could be changed to make that less nasty?